client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/MigrationDocumentationIT.java

```diff
@@ -82,7 +82,7 @@ public void testRequests() throws Exception {
         RestHighLevelClient client = highLevelClient();
         {
             //tag::migration-request-ctor
-            IndexRequest request = new IndexRequest("index", "_doc", "id"); // <1>
+            IndexRequest request = new IndexRequest("index").id("id"); // <1>
             request.source("{\"field\":\"value\"}", XContentType.JSON);
```

Comment:
Related doc says `Create an IndexRequest using its constructor` -perhaps add comment on the use of `id()` setter too.